File: android/dtos/ConfigServiceDTOs/SettingResponseDTO.kt
package net.ipv64.kivop.dtos

//
//  settings.swift
//  config-service
//
//  Created by Dennis Sept on 02.11.24.
//
public struct SettingResponseDTO: Codable{
    var id : Uuid?,
    var key : String,
    var datatype : String,
    var value : String,
    var description : String?,
)

File: DTOs/Sources/ConfigServiceDTOs/SettingResponseDTO.swift
import Foundation

public struct SettingResponseDTO: Codable{
    public var id: UUID?
    public var key: String
    public var datatype: String
    public var value: String
    public var description: String?
    
    
    public init(id: UUID? = nil, key: String, datatype: String, value: String, description: String? = nil) {
        self.id = id
        self.key = key
        self.datatype = datatype
        self.value = value
        self.description = description
    }
}

